internal/test/e2e/kafka-go/integration_test.go

```diff
@@ -39,103 +38,73 @@ func TestIntegration(t *testing.T) {
 		assert.Equal(t, "sample-app", val.AsString())
 	})
 
-	spans := ptrace.NewSpanSlice()
-	for i, scope := range scopes {
-		t.Run("Scope/"+strconv.Itoa(i), func(t *testing.T) {
-			assert.Equal(t, scopeName, scope.Scope().Name(), "scope name")
-			assert.Equal(t, semconv.SchemaURL, scope.SchemaUrl(), "schema URL")
-
-			scope.Spans().MoveAndAppendTo(spans)
-		})
-	}
-
 	// All trace ID should be the same.
-	tIDBytes := [16]byte(spans.At(0).TraceID())
+	tIDBytes := [16]byte(scopes[0].Spans().At(0).TraceID())
```

Comment:
Do we want to test here that scopes is at least of length 1?

Reply:
That is done above: https://github.com/open-telemetry/opentelemetry-go-instrumentation/pull/2310/files#diff-3848f90b432570be66c7b04a0af9ec19c43b228a964bc41ab6f0c63beccea0b2R33

Not the most obvious in my opinion, but our linting likes to use `NotEmtpy(/* ... */)` over `Len(0, spans)`. :shrug: